### ISSUE TITLE
ㅤ

### ISSUE THREAD

**Issue by ghost** [musescore/MuseScore#14368]

**Comment by Tantacrul** [comment]
Can you try installing this build and letting us know if it works for you?
https://github.com/musescore/MuseScore/suites/9417801459/artifacts/444885402

**Comment by Tantacrul** [comment]
Also, could you let us know if the Muse Hub is working any better now?

**Comment by Jeto0613** [comment]
@Tantacrul 

hello, I've tried something to launch MU4, and 2 weeks ago I succeeded to launch MU4. But the process that I launched MU4 was strange a bit and I guess it's a weird or wrong situation..

1. I found MU4 Desktop icon has administrator badge, but MU3 doesn't.
![MS](https://user-images.githubusercontent.com/117594636/203214468-98950d22-54c2-4049-8057-767656c1699f.jpg)

2. I checked my Windows account by Control Panel --> User Accounts, and my account was already Administrator.

3. I thought It's the problem in my computer, so I tested same downloading process in my dad's computer(64bit, Windows 10, 21H2). And MU4 was launched well. I checked dad's Winodows account, it was also Administrator same as mine. And I saw MU4 desktop icon has no administrator badge in dad's computer.

4. I relized, same administrators but only my account can't launch MU4.

5. I googled about windows administrator, and I set windows default administrator account active by Computer Management --> Local Users and Groups.
![MS2](https://user-images.githubusercontent.com/117594636/203217948-b8015251-4855-47ef-b712-e610234a48b3.jpg)
(sorry I'm using Korean windows)

6. I log on default administrator account and checked MU4. There was no administrator badge on desktop icon, and MU4 was launched well. 
![MS3](https://user-images.githubusercontent.com/117594636/203219256-cc6a0e0a-6c4c-4d12-89c8-3e2cee7ce7d9.jpg)
![MS4](https://user-images.githubusercontent.com/117594636/203219002-733900c3-afe7-42c4-95de-7fb92a6dd627.jpg)

So.. I found that

- MU4 can be launched by windows default administrator account only.
- same administrator account but only my computer can't launch MU4.

I can launch MU4 but I'm still confused, because there was no problem with MU3 about windows administrator..

**Comment by Jeto0613** [comment]
@Tantacrul 

> Can you try installing this build and letting us know if it works for you? https://github.com/musescore/MuseScore/suites/9417801459/artifacts/444885402

I deleted MU4 in my computer and downloaded it.

- With my user account(in administrator group), there is a same problem. MU4 loading screen appears and freezes for a few seconds, and disappears. And Muse Hub says error.
![MS5](https://user-images.githubusercontent.com/117594636/203221640-4c7ecd14-3ff0-47ad-8af2-553467d2e510.jpg)

And there is a new problem with new version.

- I can't launch MU4 even I use windows default administrator account, same problem when I launch it by use account. (loading screen appears and disappears.)

**Comment by Tantacrul** [comment]
> I downloaded the newest version by Muse Hub, and I'm using windows 11.

Unfortunately, this isn't helpful in this context. You should log it as a separate issue and give the specifics about your machine, etc. Otherwise, we'll be tourbleshooting two related - but not the same - issues in one place and confusing everyone.

**Comment by Tantacrul** [comment]
@Jeto0613 can you try:

1. Uninstalling everything (Muse Hub and all instances of MuseScore 4)
2. Next, install this directly https://github.com/musescore/MuseScore/suites/9417801459/artifacts/444885402
3. Tell us if you have any problems installing it

This is the first step in understanding the root cause of your problem. It's possibly nothing to do with the Muse Hub.

Thanks!

**Comment by Jeto0613** [comment]
@Tantacrul

3. Tell us if you have any problems opening it after install

I did these process, but there is same error.

![MS2](https://user-images.githubusercontent.com/117594636/203699831-77dbe61e-985a-490c-aea3-60d04bc67ade.JPG)
I guess.. maybe there is a problem with MU4 installer.

I downloaded MU4 by [https://ftp.osuosl.org/pub/musescore-nightlies/windows/4x/nightly/?C=M;O=D](url)
with 7z file, no installer. I downloaded 3 recent versions (lastest, 223280401, 223270402), and all those are launched well. 

There is a Microsoft Defender SmartScreen warning at the first time of launching after download nightly version.

![MS3](https://user-images.githubusercontent.com/117594636/203701314-41f2eff7-9085-4c9d-89aa-336c603e1d44.jpg)

then I turned off SmartScreen, and User Account Control, and launched installer version MU4 but it caused the same error.
